tools/UpdatePaths/Scripts/68417_mob_suit_repath.txt

```diff
@@ -0,0 +1,74 @@
+/obj/item/clothing/suit/bomb_suit : /obj/item/clothing/suit/utility/bomb_suit
+/obj/item/clothing/suit/bomb_suit/security : /obj/item/clothing/suit/utility/bomb_suit/security
+/obj/item/clothing/suit/bomb_suit/white : /obj/item/clothing/suit/utility/bomb_suit/white
+/obj/item/clothing/suit/striped_sweater : /obj/item/clothing/suit/costume/striped_sweater
+/obj/item/clothing/suit/yakuza : /obj/item/clothing/suit/costume/yakuza
+/obj/item/clothing/suit/dutch : /obj/item/clothing/suit/costume/dutch
+/obj/item/clothing/suit/irs : /obj/item/clothing/suit/costume/irs
+/obj/item/clothing/suit/osi : /obj/item/clothing/suit/costume/osi
+/obj/item/clothing/suit/tmc : /obj/item/clothing/suit/costume/tmc
+/obj/item/clothing/suit/pg : /obj/item/clothing/suit/costume/pg
+/obj/item/clothing/suit/driscoll : /obj/item/clothing/suit/costume/driscoll
+/obj/item/clothing/suit/deckers : /obj/item/clothing/suit/costume/deckers
+/obj/item/clothing/suit/morningstar : /obj/item/clothing/suit/costume/morningstar
+/obj/item/clothing/suit/saints : /obj/item/clothing/suit/costume/saints
+/obj/item/clothing/suit/phantom : /obj/item/clothing/suit/costume/phantom
+/obj/item/clothing/suit/allies : /obj/item/clothing/suit/costume/allies
+/obj/item/clothing/suit/soviet : /obj/item/clothing/suit/costume/soviet
+/obj/item/clothing/suit/yuri : /obj/item/clothing/suit/costume/yuri
+/obj/item/clothing/suit/sybil_slickers : /obj/item/clothing/suit/costume/sybil_slickers
+/obj/item/clothing/suit/basil_boys : /obj/item/clothing/suit/costume/basil_boys
+/obj/item/clothing/suit/pirate/armored : /obj/item/clothing/suit/costume/pirate/armored
+/obj/item/clothing/suit/pirate/captain : /obj/item/clothing/suit/costume/pirate/captain
+/obj/item/clothing/suit/ghost_sheet/spooky : /obj/item/clothing/suit/costume/ghost_sheet/spooky
+/obj/item/clothing/suit/det_suit/dark : /obj/item/clothing/suit/jacket/det_suit/dark
+/obj/item/clothing/suit/det_suit/noir : /obj/item/clothing/suit/jacket/det_suit/noir
+/obj/item/clothing/suit/det_suit/kim : /obj/item/clothing/suit/jacket/det_suit/kim
+/obj/item/clothing/suit/det_suit/disco : /obj/item/clothing/suit/jacket/det_suit/disco
+/obj/item/clothing/suit/bunnysuit : /obj/item/clothing/suit/costume/bunnysuit
+/obj/item/clothing/suit/security/officer/russian : /obj/item/clothing/suit/jacket/officer/tan
+/obj/item/clothing/suit/whitedress : /obj/item/clothing/suit/costume/whitedress
+/obj/item/clothing/suit/shrine_maiden : /obj/item/clothing/suit/costume/shrine_maiden
+/obj/item/clothing/suit/changshan_red : /obj/item/clothing/suit/costume/changshan_red
+/obj/item/clothing/suit/changshan_blue : /obj/item/clothing/suit/costume/changshan_blue
+/obj/item/clothing/suit/cheongsam_red : /obj/item/clothing/suit/costume/cheongsam_red
+/obj/item/clothing/suit/cheongsam_blue : /obj/item/clothing/suit/costume/cheongsam_blue
+/obj/item/clothing/suit/armor/riot/chaplain : /obj/item/clothing/suit/armor/riot/templar
```

Comment:
```suggestion
/obj/item/clothing/suit/bronze : /obj/item/clothing/suit/costume/bronze{@OLD}
/obj/item/clothing/suit/hawaiian : /obj/item/clothing/suit/costume/hawaiian{@OLD}
/obj/item/clothing/suit/ianshirt : /obj/item/clothing/suit/costume/ianshirt{@OLD}
/obj/item/clothing/suit/curator : /obj/item/clothing/suit/jacket/curator{@OLD}
/obj/item/clothing/suit/beekeeper_suit : /obj/item/clothing/suit/utility/beekeeper_suit{@OLD}
/obj/item/clothing/suit/radiation : /obj/item/clothing/suit/utility/radiation{@OLD}
/obj/item/clothing/suit/snowman : /obj/item/clothing/suit/costume/snowman{@OLD}
/obj/item/clothing/suit/pirate : /obj/item/clothing/suit/costume/pirate{@OLD}
/obj/item/clothing/suit/cardborg : /obj/item/clothing/suit/costume/cardborg{@OLD}
/obj/item/clothing/suit/fire/firefighter : /obj/item/clothing/suit/utility/fire/firefighter{@OLD}
/obj/item/clothing/suit/det_suit : /obj/item/clothing/suit/jacket/det_suit{@OLD}
/obj/item/clothing/suit/justice : /obj/item/clothing/suit/costume/justice{@OLD}
/obj/item/clothing/suit/straight_jacket : /obj/item/clothing/suit/jacket/straight_jacket{@OLD}
/obj/item/clothing/suit/cyborg_suit : /obj/item/clothing/suit/costume/cyborg_suit{@OLD}
/obj/item/clothing/suit/security/officer : /obj/item/clothing/suit/jacket/officer/blue{@OLD}
/obj/item/clothing/suit/security/warden : /obj/item/clothing/suit/jacket/warden/blue{@OLD}
/obj/item/clothing/suit/security/hos : /obj/item/clothing/suit/jacket/hos/blue{@OLD}
/obj/item/clothing/suit/nerdshirt : /obj/item/clothing/suit/costume/nerdshirt{@OLD}
/obj/item/clothing/suit/xenos : /obj/item/clothing/suit/costume/xenos{@OLD}
/obj/item/clothing/suit/monkeysuit : /obj/item/clothing/suit/costume/monkeysuit{@OLD}
/obj/item/clothing/suit/judgerobe : /obj/item/clothing/suit/costume/judgerobe{@OLD}
/obj/item/clothing/suit/chickensuit : /obj/item/clothing/suit/costume/chickensuit{@OLD}
/obj/item/clothing/suit/ghost_sheet : /obj/item/clothing/suit/costume/ghost_sheet{@OLD}
/obj/item/clothing/suit/imperium_monk : /obj/item/clothing/suit/costume/imperium_monk{@OLD}
/obj/item/clothing/suit/poncho : /obj/item/clothing/suit/costume/poncho{@OLD}
/obj/item/clothing/suit/poncho/green : /obj/item/clothing/suit/costume/poncho/green{@OLD}
/obj/item/clothing/suit/poncho/red : /obj/item/clothing/suit/costume/poncho/red{@OLD}
/obj/item/clothing/suit/armor/plate/crusader/red : /obj/item/clothing/suit/chaplainsuit/crusader/red{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain/clock : /obj/item/clothing/suit/chaplainsuit/clock{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain : /obj/item/clothing/suit/chaplainsuit/templar{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain/studentuni : /obj/item/clothing/suit/chaplainsuit/studentuni{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain/ancient : /obj/item/clothing/suit/chaplainsuit/ancient{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain/witchhunter : /obj/item/clothing/suit/chaplainsuit/witchhunter{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain/adept : /obj/item/clothing/suit/chaplainsuit/adept{@OLD}
/obj/item/clothing/suit/capjacket : /obj/item/clothing/suit/jacket/capjacket{@OLD}
/obj/item/clothing/suit/fire/atmos : /obj/item/clothing/suit/utility/fire/atmos{@OLD}
/obj/item/clothing/suit/bomb_suit : /obj/item/clothing/suit/utility/bomb_suit{@OLD}
/obj/item/clothing/suit/bomb_suit/security : /obj/item/clothing/suit/utility/bomb_suit/security{@OLD}
/obj/item/clothing/suit/bomb_suit/white : /obj/item/clothing/suit/utility/bomb_suit/white{@OLD}
/obj/item/clothing/suit/striped_sweater : /obj/item/clothing/suit/costume/striped_sweater{@OLD}
/obj/item/clothing/suit/yakuza : /obj/item/clothing/suit/costume/yakuza{@OLD}
/obj/item/clothing/suit/dutch : /obj/item/clothing/suit/costume/dutch{@OLD}
/obj/item/clothing/suit/irs : /obj/item/clothing/suit/costume/irs{@OLD}
/obj/item/clothing/suit/osi : /obj/item/clothing/suit/costume/osi{@OLD}
/obj/item/clothing/suit/tmc : /obj/item/clothing/suit/costume/tmc{@OLD}
/obj/item/clothing/suit/pg : /obj/item/clothing/suit/costume/pg{@OLD}
/obj/item/clothing/suit/driscoll : /obj/item/clothing/suit/costume/driscoll{@OLD}
/obj/item/clothing/suit/deckers : /obj/item/clothing/suit/costume/deckers{@OLD}
/obj/item/clothing/suit/morningstar : /obj/item/clothing/suit/costume/morningstar{@OLD}
/obj/item/clothing/suit/saints : /obj/item/clothing/suit/costume/saints{@OLD}
/obj/item/clothing/suit/phantom : /obj/item/clothing/suit/costume/phantom{@OLD}
/obj/item/clothing/suit/allies : /obj/item/clothing/suit/costume/allies{@OLD}
/obj/item/clothing/suit/soviet : /obj/item/clothing/suit/costume/soviet{@OLD}
/obj/item/clothing/suit/yuri : /obj/item/clothing/suit/costume/yuri{@OLD}
/obj/item/clothing/suit/sybil_slickers : /obj/item/clothing/suit/costume/sybil_slickers{@OLD}
/obj/item/clothing/suit/basil_boys : /obj/item/clothing/suit/costume/basil_boys{@OLD}
/obj/item/clothing/suit/pirate/armored : /obj/item/clothing/suit/costume/pirate/armored{@OLD}
/obj/item/clothing/suit/pirate/captain : /obj/item/clothing/suit/costume/pirate/captain{@OLD}
/obj/item/clothing/suit/ghost_sheet/spooky : /obj/item/clothing/suit/costume/ghost_sheet/spooky{@OLD}
/obj/item/clothing/suit/det_suit/dark : /obj/item/clothing/suit/jacket/det_suit/dark{@OLD}
/obj/item/clothing/suit/det_suit/noir : /obj/item/clothing/suit/jacket/det_suit/noir{@OLD}
/obj/item/clothing/suit/det_suit/kim : /obj/item/clothing/suit/jacket/det_suit/kim{@OLD}
/obj/item/clothing/suit/det_suit/disco : /obj/item/clothing/suit/jacket/det_suit/disco{@OLD}
/obj/item/clothing/suit/bunnysuit : /obj/item/clothing/suit/costume/bunnysuit{@OLD}
/obj/item/clothing/suit/security/officer/russian : /obj/item/clothing/suit/jacket/officer/tan{@OLD}
/obj/item/clothing/suit/whitedress : /obj/item/clothing/suit/costume/whitedress{@OLD}
/obj/item/clothing/suit/shrine_maiden : /obj/item/clothing/suit/costume/shrine_maiden{@OLD}
/obj/item/clothing/suit/changshan_red : /obj/item/clothing/suit/costume/changshan_red{@OLD}
/obj/item/clothing/suit/changshan_blue : /obj/item/clothing/suit/costume/changshan_blue{@OLD}
/obj/item/clothing/suit/cheongsam_red : /obj/item/clothing/suit/costume/cheongsam_red{@OLD}
/obj/item/clothing/suit/cheongsam_blue : /obj/item/clothing/suit/costume/cheongsam_blue{@OLD}
/obj/item/clothing/suit/armor/riot/chaplain : /obj/item/clothing/suit/armor/riot/templar{@OLD}
```

You want to have `{@OLD}` on all of the paths you replace, otherwise UpdatePaths will discard things like pixel-shifting, name edits, or description edits in maps (AKA: years of soul will be removed if this script is run as-is and commited). Your manual update done earlier won't do this and that's fine, but this is important to ensure we aren't needlessly removing work.

Reply:
<3